configs/proxy_connect_udp.yaml

```diff
@@ -0,0 +1,59 @@
+# This will proxy CONNECT-UDP requests from a downstream connecting on 127.0.0.1:10001 to an
+# upstream listening on 127.0.0.1:10002. Note that the downstream client uses TCP in this example.
+admin:
+  address:
+    socket_address:
+      protocol: TCP
+      address: 127.0.0.1
+      port_value: 9902
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        protocol: TCP
+        address: 127.0.0.1
+        port_value: 10001
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
```

Comment:
Maybe this is an ignorant question, but is it common to use googleapis.com in Envoy configs?

Reply:
Yes, it is the actual name of the type.